.github/workflows/acceptance_tests_gke.yaml

```diff
@@ -11,32 +11,39 @@ on:
         default: us-east1-b
       kubernetesVersion:
         description: The GKE kubernetes version
-        default: 1.23
+        default: 1.24
       workersCount:
         description: The number of cluster nodes to provision
-        default: 2
+        default: 10
```

Comment:
Just wondering how you arrived at 10 x e2-standard-8? What about a smaller number of beefier machines?